src/Microsoft.Health.Fhir.Stu3.Web/Microsoft.Health.Fhir.Stu3.Web.csproj

```diff
@@ -11,6 +11,9 @@
     <PackageReference Include="Microsoft.Azure.Services.AppAuthentication" Version="1.3.1" />
     <PackageReference Include="Microsoft.Extensions.Configuration.AzureKeyVault" Version="3.1.3" />
     <PackageReference Include="Microsoft.Extensions.Logging.ApplicationInsights" Version="2.12.0" />
+    <PackageReference Include="prometheus-net.AspNetCore" Version="3.6.0" />
```

Comment:
Should this be added to https://github.com/microsoft/fhir-server/blob/master/THIRDPARTYNOTICES.md?